### ISSUE TITLE
UTF8::encode('UTF-8', $str) is not enconding the character "á" correctly

### ISSUE THREAD

**Issue by gabriel-silveira** [voku/portable-utf8#85]
I have strings in ISO-8859-1 and Win-1252 and when a i try to encode to UTF-8 using the encode method all is doing well except for the "á" character that is not encoding correctly.
What it could be?

```
$str1 = file_get_contents('../encoding_detect/1/iso-8859-1.html');
print(UTF8::encode('UTF-8', $str1).'<br/>');
$str2 = file_get_contents('../encoding_detect/1/win-1252.html');
print(UTF8::encode('UTF-8', $str2).'<br/>');
```

**Comment by voku** [comment]
Can you send me the test file, or even better can you create a pull request with a failing test?

**Comment by gabriel-silveira** [comment]
Thanks for your fast response! The page in the url is in Win1252, but the str_detect_encoding is returning ISO-8859-1. The encode method is not encoding correctly the "á", like i said.
Below is an example of code that illustrate the issue:
```
<?php
require __DIR__ . '/vendor/autoload.php';

use voku\helper\UTF8;

$str = file_get_contents('http://www.gabrielsilveira.com.br/encodings/1252.html');

$encoding = UTF8::str_detect_encoding($str);
print($encoding.'<br/>');
print(UTF8::encode('UTF-8', $str));
```

**Comment by voku** [comment]
Thanks for the test file, fixed in version 5.4.8

Happy coding :+1:

**Comment by gabriel-silveira** [comment]
Thank you for your effort. You really did a great project! Congratulations, @voku !
